siliconcompiler/core.py

```diff
@@ -2391,7 +2396,7 @@ def _collect(self, directory=None):
                 self.error(f'Failed to copy {path}', fatal=True)
 
     ###########################################################################
-    def archive(self, step=None, index=None, all_files=False):
+    def archive(self, step=None, index=None, nodelist=None, all_files=False, archive_name=None, include_manifest=True, additional_files=None):
```

Comment:
Why introduce `nodelist` as a param if the testcase generator only passes in a single step/index anyway?

Reply:
removed